scripts/us_cdc/environmental_health_toxicology/README.md

```diff
@@ -47,80 +47,101 @@ These data were collected as part of the [CDC National Environment Public Health
+[`Ozone_Daily_County_test_file.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/air_quality/test_files/Ozone_Daily_County_test_file.csv)
 
-[`small_Ozone_County_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/test_data/small_Ozone_County_expected.csv)
+[`Ozone_Daily_County_test_file_expected_output.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/air_quality/test_files/Ozone_Daily_County_test_file_expected_output.csv)
 
-[`small_Palmer.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/test_data/small_Palmer.csv)
+[`PM2.5_Concentrations_Daily_Census_Tract_test_file.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/air_quality/test_files/PM2.5_Concentrations_Daily_Census_Tract_test_file.csv)
 
-[`small_Palmer_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/test_data/small_Palmer_expected.csv)
+[`PM2.5_Concentrations_Daily_Census_Tract_test_file_expected_output.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/air_quality/test_files/PM2.5_Concentrations_Daily_Census_Tract_test_file_expected_output.csv)
 
 #### tMCFs
-[`OzoneCensusTractPollution.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/OzoneCensusTractPollution.tmcf)
+[`OzoneCensusTractPollution.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/air_quality/tMCFs/OzoneCensusTractPollution.tmcf)
+
+[`OzoneCountyPollution.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/air_quality/tMCFs/environmental_health_toxicology/air_quality/tMCFs/OzoneCountyPollution.tmcf)
+
+[`PM25CensusTractPollution.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/air_quality/environmental_health_toxicology/air_quality/tMCFs/PM25CensusTractPollution.tmcf)
+
+[`PM25CountyPollution.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/air_quality/tMCFs/PM25CountyPollution.tmcf)
+
+### Precipitation Artifacts
+
+#### Script
+[`parse_precipitation_index.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/precipitation/scripts/parse_precipitation_index.py)
+
+[`parse_precipitation_index_test.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/precipitation/scripts/parse_precipitation_index_test.py)
 
-[`OzoneCountyPollution.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/OzoneCountyPollution.tmcf)
+#### Test Files
```

Comment:
This is fine now, but for future, not necessary to list test artifacts.

Reply:
got it